migrations/2023-07-26-222023_site-aggregates-one/up.sql

```diff
@@ -0,0 +1,15 @@
+create or replace function site_aggregates_site()
+returns trigger language plpgsql
+as $$
+begin
+  -- we only ever want to have a single value in site_aggregate because the site_aggregate triggers update all rows in that table.
+  -- a cleaner check would be to insert it for the local_site but that would break assumptions at least in the tests
+  IF (TG_OP = 'INSERT') AND NOT EXISTS (select id from site_aggregates limit 1) THEN
```

Comment:
Maybe better than replacing this function, would be to change the trigger, so that this function never runs for non-local sites in the first place.

IE change the trigger to only fire `when NEW.site_id = 1`

Reply:
i wanted to do that, but the problem is that for the site tests that assumption doesn't hold - the site_id can be something different than 1

Reply:
Maybe the real issue here is that since site_aggregates should only exist for the local_site anyway, everything should be reworked to join / trigger not off the site table, but local_site. 

We can leave that for a later PR, but i'll open up an issue.

Reply:
Why is the site counting person scores for persons from other instances? It doesn't even account for the communities that the local instance has no comment, post, and votes for (no subscribers). Same with remote communities.

Reply:
So that front ends can have predictable `CommunityView` and `SiteView`s. But that could be re-evaluated.